config/routes.rb

```diff
@@ -1,3 +1,22 @@
 Rails.application.routes.draw do
-  # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
-end
+  #### Restaurants ####
+  resources :restaurants
+
+  #### Reviews ####
```

Comment:
I think we should rewrite this bit - in the "rails way" using the functions we get like `resources` and giving it a block
https://guides.rubyonrails.org/routing.html#nested-resources